cmd/podman/run_test.go

```diff
@@ -66,19 +66,28 @@ func createCLI() cli.App {
 	return a
 }
 
-func getRuntimeSpec(c *cli.Context) *spec.Spec {
+func getRuntimeSpec(c *cli.Context) (*spec.Spec, error) {
 	runtime, _ := getRuntime(c)
```

Comment:
Should there be an error check here as well?

Reply:
Yup pushed a new version.

Reply:
It seems like getRuntime() is erroring, which means we never got a non-nil runtime before the error check was added. Can we replace runtime with nil in the call below?

Reply:
Theoretically the call below could crash if runtime is nil.  It uses runtime, if you are joining a container to another container namespace.  I guess we don't test that now.